Formula/libmf.rb

```diff
@@ -0,0 +1,41 @@
+class Libmf < Formula
+  desc "open source tool for approximating an incomplete matrix."
+  homepage "https://www.csie.ntu.edu.tw/~cjlin/libmf/"
+  url "http://www.csie.ntu.edu.tw/~cjlin/libmf/libmf-2.01.zip"
+  sha256 "75bb98a4e8f1a318d7d07556583727b4e301759904570bf527972d964d34ef30"
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!